### ISSUE TITLE
2.11.2 broken with nodebb

### ISSUE THREAD

**Issue by max06** [ai/browserslist#212]
Good morning,

I just noticed that the package was updated, since then my nodebb template builds are failing. I can't tell why and how, just ensured that the previous version (2.11.1) works without problems.

You might want to have a look on that.

**Comment by ai** [comment]
What is exactly the error?

How do you use Browserslist in nodebb? Do you use it on slide-side and bundle to client-side JS with webpack/browserify?

**Comment by max06** [comment]
Good question. Nodebb uses "autoprefixer" which then includes your package. 
I'm not a developer of the nodebb project, I sadly have no clue how it's used in there.

**Comment by ai** [comment]
You said you have some problem. What exactly problem did you have? What you execute and what result got?

**Comment by max06** [comment]
checkout nodebb and run ./nodebb build - that one failed for me without a proper error message. After that all minified js-bundles and templates were missing. I figured out that your update caused the issue by comparing the package lock file between a version that worked 3 days ago and a version that was built some minutes ago. 

Shell output:
```
15/1 09:43:27 [13551] - info: [build] Building in parallel mode
15/1 09:43:27 [13551] - info: [build]         plugin static dirs  build started
15/1 09:43:27 [13551] - info: [build]          requirejs modules  build started
15/1 09:43:27 [13551] - info: [build]           client js bundle  build started
15/1 09:43:27 [13551] - info: [build]            admin js bundle  build started
15/1 09:43:28 [13551] - info: [build]         client side styles  build started
15/1 09:43:28 [13551] - info: [build] admin control panel styles  build started
15/1 09:43:28 [13551] - info: [build]                  templates  build started
15/1 09:43:28 [13551] - info: [build]                  languages  build started
15/1 09:43:28 [13551] - info: [build]                     sounds  build started
15/1 09:43:28 [13551] - info: [build]                     sounds  build completed in 0.431sec
15/1 09:43:28 [13551] - info: [build]         plugin static dirs  build completed in 0.65sec
15/1 09:43:30 [13551] - info: [build]                  templates  build completed in 2.866sec
15/1 09:43:31 [13551] - error: [build] admin control panel styles  build failed
15/1 09:43:31 [13551] - error: [build] Encountered error during build step: undefined
15/1 09:43:31 [13551] - error:
```

**Comment by ai** [comment]
Open an issue in `nodebb`, they definitely should have better error reports.

Let’s wait for better error details.

**Comment by max06** [comment]
You're absolutely right about that. In the meanwhile I worked around the issue by pinning your package back to 2.11.1

**Comment by max06** [comment]
Issue opened here: https://github.com/NodeBB/NodeBB/issues/6241

**Comment by Andarist** [comment]
It seems that this probably also fails babel at the moment [here](https://github.com/babel/babel/blob/79c84f2f9bcb74319a89c4da3328b9d9b1a68c65/packages/babel-preset-env/src/targets-parser.js#L118) with:
```
1) getTargets parses:
     TypeError: Path must be a string. Received undefined
      at assertPath (path.js:28:11)
      at Object.resolve (path.js:1181:7)
      at Object.findConfig (packages/babel-preset-env/node_modules/browserslist/node.js:163:17)
      at Object.loadConfig (packages/babel-preset-env/node_modules/browserslist/node.js:126:37)
      at browserslist (packages/babel-preset-env/node_modules/browserslist/index.js:194:22)
      at getTargets (packages/babel-preset-env/src/targets-parser.js:118:22)
      at Context.<anonymous> (packages/babel-preset-env/test/targets-parser.spec.js:7:7)
```

**Comment by ai** [comment]
Thanks. Wait a second, I will fix it.

**Comment by Andarist** [comment]
Cool! Thanks.

**Comment by ai** [comment]
Fixed https://github.com/ai/browserslist/commit/9490380fc5d4fb67d2fdfec62380d9b8996eae4b

**Comment by ai** [comment]
Released in 2.11.3.

**Comment by max06** [comment]
Awesome. I can confirm that this fix also solved my issue here :+1:
Sorry for being so unspecific.
